curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-introductory-javascript-by-building-a-pyramid-generator/6610c71600966a2191d3a64a.md

```diff
@@ -7,17 +7,14 @@ dashedName: step-52
 
 # --description--
 
-There is a bit of a problem, here. Your `test` declaration is currently after your `return` statement. An important thing to know about the `return` keyword is that it does not just define a value to be returned from your function, it also stops the execution of your function code. This means that any code after a `return` statement will not run.
-
-Move your `test` initialization to the line above your `return` statement.
+Returning a value from a function brings that value into the scope where the function was called. To bring your `"Testing"` value from the `padRow` function into the global scope, update your return statement to return only the `test` variable.
```

Comment:
````md
Values returned out of functions are used by calling the function. You can use the function call directly as the value it returns, or capture the returned value in a variable. This way, you can use the value assigned to a locally scoped variable, outside the function it was created in.

```js
function getName() {
  const name = "Camper cat";
  return name;
}

console.log(getName()); // "Camper cat"

const capturedReturnValue = getName();
console.log(capturedReturnValue); // "Camper cat"

console.log(name); // reference error
```

To use your `"Testing"` value returned from the `padRow` function, update your `return` statement to return only the `test` variable.

````

---

This is just a suggestion, but I think what we have now can be misinterpreted in multiple ways. I suggest we simplify this and give a small code example.